client/src/partials/purchases/create/templates/code.tmpl.html

```diff
@@ -0,0 +1,15 @@
+<div style="padding : 5px;">
```

Comment:
I'm not sure about this one using the default cell class with the input and expected padding - if you could verify if this will work with the class and if not there is no issue here.